source/backend/api/Services/ProjectService.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Security.Claims;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Entities;
+using Pims.Dal.Repositories;
+
+namespace Pims.Api.Services
+{
+    public class ProjectService : BaseService, IProjectService
+    {
+        private readonly ILogger _logger;
+        private readonly IProjectRepository _projectRepository;
+
+
+        /// <summary>
+        /// Creates a new instance of a ProjectService, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        /// <param name="projectRepository"></param>
+        public ProjectService(
+            ClaimsPrincipal user,
+            ILogger<ProjectService> logger,
+            IProjectRepository projectRepository)
+            : base(user, logger)
+        {
+            _logger = logger;
+            _projectRepository = projectRepository;
+        }
+
+        public IList<PimsProject> GetProjectPredictions(string filter, int maxResult)
```

Comment:
Can you explain the use of predictions here? Are we using this somewhere else maybe? This doesn't seem to describe search functionality but maybe I don't understand the context.

Reply:
Well, basically I used the same ideas used by the person/organization autocomplete. It could be renamed to search/filter or anything else to make it more reusable.

Reply:
Ok, if that is already in use this is fine.